src/services/api.ts

```diff
@@ -14,10 +14,10 @@ export async function fetchTopManga(): Promise<RankDetails[]> {
   return response.data.rank;
 }
 
-// Call the API to get the top/trending manwha.
-export async function fetchTopManwha(): Promise<RankDetails[]> {
+// Call the API to get the top/trending manhwa.
+export async function fetchTopManhwa(): Promise<RankDetails[]> {
   const response = await axios.get(
-    "https://api.comick.fun/top?type=trending&comic_types=manwha&accept_mature_content=false"
+    "https://api.comick.fun/top?type=trending&comic_types=manhwa&accept_mature_content=false"
```

Comment:
I am not sure if you mentioned this in your PR description, but try to place the base URL in it's own file and import it wherever you need it. That way when it changes you won't have to change it everywhere.

Reply:
I have added a variable called ```baseURL```  to ```api.ts``` and then added this to the API call each time instead of repeating it.